src/libraries/Directory.Build.props

```diff
@@ -5,20 +5,18 @@
   <Import Project="..\..\Directory.Build.props" />
 
   <ItemGroup>
-    <PortedExtensionsProject Include="$(MSBuildThisFileDirectory)Microsoft.Extensions.Caching.Abstractions\**\*csproj" />
```

Comment:
Perhaps now is the time to delete all the exclusions and instead just exclude the couple tests that aren't yet done.

Reply:
Agreed :)